recipes/implot/all/test_package/conanfile.py

```diff
@@ -1,15 +1,31 @@
-from conans import ConanFile, CMake, tools
 import os
 
-class TestPackageConan(ConanFile):
+from conan import ConanFile
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.build import can_run
+
+
+class ImplotTestConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake", "cmake_find_package_multi"
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
+
+    def generate(self):
+        deps = CMakeDeps(self)
+        deps.generate()
+        tc = CMakeToolchain(self)
+        tc.generate()
```

Comment:
```suggestion
```
Maybe using generators atribute would be less verbose (see https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/test_package/conanfile.py#L10)

Reply:
You are totally right. This is a relict of my experiments.

However, I don't think "VirtualRunEnv" is needed. At least `conan new cmake_lib` doesn't specify it.

Reply:
Done in [48c2153](https://github.com/conan-io/conan-center-index/pull/16847/commits/48c2153b35676b4d8db6cf120c8b358fe1d86a46)

Reply:
Ok. Might be true.

Done in [dc59bf4](https://github.com/conan-io/conan-center-index/pull/16847/commits/dc59bf4ee8cca0ef611fc035fcac30c4faf5c879)

Reply:
This is because test_packge is ran by v1 and v2 pipeline and the Conan 1.x does not setup the Venvs by default so we need to explicit set them for the mto woork here :)

(my typing died)